src/array/array.cc

```diff
@@ -679,5 +717,17 @@ DGL_REGISTER_GLOBAL("ndarray._CAPI_DGLCreateSparseMatrix")
     *rv = SparseMatrixRef(spmat);
   });
 
+DGL_REGISTER_GLOBAL("geometry._CAPI_FarthestPointSampler")
+.set_body([] (DGLArgs args, DGLRetValue* rv) {
+    const NDArray data = args[0];
+    const int64_t batch_size = args[1];
+    const int64_t sample_points = args[2];
+    NDArray dist = args[3];
+    IdArray start_idx = args[4];
+    IdArray result = args[5];
+    const DLDeviceType XPU = data->ctx.device_type;
+    FarthestPointSampler(data, batch_size, sample_points, dist, start_idx, result);
+  });
+
```

Comment:
So here are two questions. 
1. Should we put it in `src/array` or in a separate package? @jermainewang 's opinion is to put it into array, as there won't be too many related ops and then the single-out part will just be a mini package.
2. If to be singled out, what is the name of this package? `point_cloud` sounds too narrow as it might be related to 3d applications in other data formats.

Reply:
Having minipackages sounds better to me as the code would be easier to navigate in e.g. source browsers.  If we put the geometry-specific code in `array`, soon we will have lots of domain-specific kernels in `array` when we extend to other domains and the code will be much harder to maintain.

If `point_cloud` sounds too narrow then how about `geometry`?

I would say that the `geometry_op_impl` files should be in the `geometry` minipackage as well.

Reply:
I can sure make the change if all reviewers agree. For naming, `geometry` sounds good to me (obviously).